ci-operator/step-registry/proxy/whitelist/enable/proxy-whitelist-enable-commands.sh

```diff
@@ -104,7 +105,7 @@ EOF
 
 # for nightly test
 cat <<EOF >> ${SHARED_DIR}/proxy_whitelist.txt
-registry.ci.openshift.org
+.ci.openshift.org
```

Comment:
What case need to change this one ?

Reply:
@jianlinliu I checked the current aws-ipi-proxy-whitelist job, such as
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/periodic-ci-openshift-verification-tests-master-installation-nightly-4.16-aws-ipi-proxy-whitelist-arm-f14/1811648697822351360, it's failing for `registry.build03.ci.openshift.org` couldn't be visited, so in prow, it's not `registry.ci.openshift.org`, and not always  `registry.build03.ci.openshift.org`.

Reply:
Got it.